src/main/java/org/opentripplanner/middleware/triptracker/TripInstruction.java

```diff
@@ -121,4 +151,35 @@ private String buildOnTrackInstruction() {
         }
         return NO_INSTRUCTION;
     }
+
+    /**
+     * Build wait for bus instruction.
+     */
+    private String buildWaitForBusInstruction() {
+        String routeId = busLeg.route.id.split(":")[1];
+        if (busLeg.departureDelay > 0) {
+            long waitInMinutes = Duration
+                .between(currentTime.atZone(DateTimeUtils.getOtpZoneId()), busLeg.getScheduledStartTime())
+                .toMinutes();
+            return String.format("Wait%s for bus %s", getReadableMinutes(waitInMinutes), routeId);
+        } else {
+            return String.format(
+                "Wait for bus %s scheduled to arrive at %s",
+                routeId,
+                BUS_OPERATOR_NOTIFIER_API_TIME_FORMAT.format(busLeg.getScheduledStartTime())
```

Comment:
For the instruction, use the existing time formatting code from #201 that supports i18n and doesn't show seconds.